internal/compiler/tests/syntax/elements/menubar2.slint

```diff
@@ -19,33 +19,48 @@ export component A inherits Window {
         init => {}
 //      ^error{'init' is not a callback in MenuBar}
 
-        MenuItem {
+        Menu {
             title: "hello";
             x: 0px;
-//          ^error{Unknown property x in MenuItem}
+//          ^error{Unknown property x in Menu}
             max-height: 13px;
-//          ^error{Unknown property max-height in MenuItem}
+//          ^error{Unknown property max-height in Menu}
             opacity: 0.4;
-//          ^error{Unknown property opacity in MenuItem}
+//          ^error{Unknown property opacity in Menu}
             Rectangle {
-//          ^error{Rectangle is not allowed within MenuItem. Only MenuItem are valid children}
+//          ^error{Rectangle is not allowed within Menu. Only Menu MenuItem are valid children}
             }
 
             MenuItem {
                 activated => {
                         mb.activated({});
 //                         ^error{Element 'MenuBar' does not have a property 'activated'}
                 }
+
+                TouchArea {}
+//              ^error{MenuItem cannot have children elements}
             }
 
         }
+
+        MenuItem {}
+//      ^error{MenuItem is not allowed within MenuBar. Only Menu are valid children}
     }
 
     Rectangle {
         x: 45px;
         MenuBar {
 //      ^error{MenuBar can only be within a Window element}
         }
+
+        MenuItem {
+//      ^error{Unknown element 'MenuItem'}
+            Menu {}
+//          ^error{Menu can only be within a ContextMenuArea element}
```

Comment:
There error are not ideal as they don't mention MenuBar or that you can have MenuItem within Menu. But that's because the that report this error don't check for additional elements in element that cannot themselves be used everywhere. This can be fixed later.

Reply:
Agreed, this can be fixed later.